docs/calendar.md

```diff
@@ -18,7 +18,7 @@ Some places use another calendar system as the main calendar, or have a separate
 > **NOTE:** The Temporal polyfill currently only includes the ISO 8601 calendar, but the Temporal proposal will eventually provide all the calendars supported by Intl.
 > See [issue #541](https://github.com/tc39/proposal-temporal/issues/541).
 
-### When to use the `Temporal.Calendar`
+### When to use `Temporal.Calendar`
```

Comment:
Thanks. This has been annoying me for a while, but I keep forgetting to fix it. :rofl: